src/providers/BackgroundBlurProvider/docs/BackgroundBlurProvider.stories.mdx

```diff
@@ -27,13 +27,16 @@ when mounted, which may impact performance.
 
 You should see either "processor is supported" or "processor is not supported" in your browser developer tools based on whether or not
 background blur is supported on your device and browser version. For more information on if background blur is supported, refer
-to [Amazon Chime JS SDK Background Blur Guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/15_Background_Filter_Video_Processor.md#integrating-background-blur-into-your-amazon-chime-sdk-for-javascript-application).
+to [Amazon Chime JS SDK Background Blur Guide](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/15_Background_Filter_Video_Processor.md#integrating-background-filters-into-your-amazon-chime-sdk-for-javascript-application).
 
-You should check whether or not the processor has been loaded correctly by checking the state of `isBackgroundBlurSupported` before calling `createBackgroundBlurDevice`.
-`createBackgroundBlurDevice` may throw an error if the processor has not been loaded yet. 
+You can check whether or not the processor has been loaded correctly by checking the state of `isBackgroundBlurSupported`.
+`createBackgroundBlurDevice` may throw an error if the processor was not loaded. You should check whether or not the processor has been loaded correctly by checking the state of `isBackgroundBlurSupported`
+before calling `createBackgroundBlurDevice`. Calling `createBackgroundBlurDevice` will create a new processor. Users would also need to call DefaultVideoTransformDevice.stop when constructing a new 
+DefaultVideoTransformDevice with new video processors. For more information, refer to [Video Processing APIs](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/10_Video_Processor.md#stopping-videotransformdevice);
 
 One thing to note is that calling `meetingManager.selectVideoInputDevice()` with a `Device` type while the current selected video input device is a `VideoTransformDevice`
-will automatically stop any video processor that was previously running.
+will automatically stop any video processor that was previously running. Lastly, make sure to construct a new DefaultVideoTransformDevice using `createBackgroundBlurDevice` 
```

Comment:
looking into if this step can be done using hooks without having the builder re create a device but rather listen to hook event change.